include/preprocess/max_inscribed_ellipsoid_rounding.hpp

```diff
@@ -44,18 +45,28 @@ std::tuple<MT, VT, NT> max_inscribed_ellipsoid_rounding(Polytope &P,
         E = (E + E.transpose()) / 2.0;
         E = E + MT::Identity(d, d)*std::pow(10, -8.0); //normalize E
 
-        Eigen::LLT<MT> lltOfA(E); // compute the Cholesky decomposition of E
+        Eigen::LLT<MT> lltOfA(E.inverse()); // compute the Cholesky decomposition of E^{-1}
```

Comment:
Is `E` psd? If so why not invert it more efficiently e.g. `E.llt().solve(I)`?

Reply:
Thanks, done